lib/dfe/reference_data/degrees/institutions.rb

```diff
@@ -2376,17 +2388,20 @@ module Degrees
             ukprn: '10020381',
             'has_never_awarded_degrees' => true },
           '203f182c-1425-ec11-b6e6-000d3adf095a' =>
-          { name: 'UA92',
+          { name: 'University Academy 92',
```

Comment:
Any reason for switching to the long name? Looks like their main brand is UA92? https://ua92.ac.uk

Reply:
It looks like both are used on the website - though possibly `UA92` a bit more commonly. My thinking here was that the abbreviation is mildly ambiguous (moreso for non UA92 people) so spelling it out might be more explicit. We don't have to make the change though.